test/unit/theory/theory_bv_opt_white.cpp

```diff
@@ -151,5 +151,31 @@ TEST_F(TestTheoryWhiteBVOpt, signed_max)
             << std::endl;
 }
 
+TEST_F(TestTheoryWhiteBVOpt, min_boundary)
+{
+  Node x = d_nodeManager->mkVar(*d_BV32Type);
+  // do not remove
```

Comment:
Is this needed? If not, please remove. Otherwise, make the comment clearer.

Reply:
It's described in the following statement. just don't remove it.

Reply:
This comment is not necessary. If the statement would be removed it would result in a compile error. The main point here is that line 165 should not be removed/disabled since this will trigger the described bug.

Reply:
you mean the comment? ok I gonna remove